storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/PersonGetResolver.java

```diff
@@ -0,0 +1,39 @@
+package com.pushtorefresh.storio.sample.db.resolvers;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.sample.db.entities.Car;
+import com.pushtorefresh.storio.sample.db.entities.Person;
+import com.pushtorefresh.storio.sample.db.tables.CarsTable;
+import com.pushtorefresh.storio.sample.db.tables.UsersTable;
+import com.pushtorefresh.storio.sqlite.operations.get.DefaultGetResolver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonGetResolver extends DefaultGetResolver<Person> {
+
+    // We expect that cursor will contain both Tweet and User: SQL JOIN
+    @NonNull
+    @Override
+    public Person mapFromCursor(@NonNull Cursor cursor) {
+        final Person person = Person.newPerson(
+                cursor.getLong(cursor.getColumnIndexOrThrow(UsersTable.COLUMN_ID)),
+                cursor.getString(cursor.getColumnIndexOrThrow(UsersTable.COLUMN_NICK))
+        );
+
+        List<Car> cars = new ArrayList<Car>();
+        while (cursor.moveToNext()) {
```

Comment:
> Solution 2 is an outer join, right?

Nope, Solution 2 is a separate query to the `cars` table for each `person`, your code is correct :)

To get `storIOSQLite` instance you need to override `GetResolver` instead of `DefaultGetResolver` (Default resolvers were made for simplicity, but you always can override most abstract resolvers to write any kind of custom code :)